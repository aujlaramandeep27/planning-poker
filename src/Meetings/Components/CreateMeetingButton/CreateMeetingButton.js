import React from 'react';
import {connect} from 'react-redux';
import {viewCreateMeeting} from '../../../Navigation/route-actions';

export function CreateMeetingButton(props) {
    return (
        <div className="uk-card uk-card-secondary uk-card-body pp-secondary-button" onClick={event => GoToCreateMeeting(event)}>
            <h3 className="uk-card-title">+</h3>
        </div>
    );

    function GoToCreateMeeting(event) {
        event.preventDefault();
        props.goToCreateMeeting();
    }
}
  
function mapStateToProps(state) {
    return {}
};

 function mapDispatchToProps(dispatch) {
    return {
        goToCreateMeeting: () => dispatch(viewCreateMeeting())
    }
};
 
export default connect(mapStateToProps, mapDispatchToProps)(CreateMeetingButton)