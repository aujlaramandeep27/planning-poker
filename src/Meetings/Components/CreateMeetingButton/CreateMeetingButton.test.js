import CreateMeetingButton from './CreateMeetingButton';

it('AddMeetingButton renders correctly', () => {
    expect(CreateMeetingButton()).toMatchSnapshot(); 
});