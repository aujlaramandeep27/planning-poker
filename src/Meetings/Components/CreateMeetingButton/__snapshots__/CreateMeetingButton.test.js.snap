// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMeetingButton renders correctly 1`] = `
<div
  className="uk-card uk-card-secondary uk-card-body pp-secondary-button"
  onClick={[Function]}
>
  <h3
    className="uk-card-title"
  >
    +
  </h3>
</div>
`;
